frontend/src/metabase/writeback/containers/ActionParametersInputForm/utils.unit.spec.ts

```diff
@@ -0,0 +1,146 @@
+import type { FieldSettingsMap } from "metabase-types/api";
+import { formatParametersBeforeSubmit, setDefaultValues } from "./utils";
+
+describe("writeback > containers > ActionParametersInputForm > utils", () => {
+  describe("formatParametersBeforeSubmit", () => {
```

Comment:
Thank you sooo much for covering all that! ❤️

Reply:
copilot wrote the this test actually 😂  it's really good at writing simple tests.